### DISCUSSION TITLE
AEA8D485-8C2A-4F04-9E37-D7516448D3CB

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#103] [General]
# Alpha Arietis

Alpha Arietis, also known as Hamal, is an K-type giant star only about 65.88 light-years away in the constellation Aries. It is the brightest visible star in the said constellation, and has a large gas giant planet that orbits roughly 1.5 times the distance of the Earth from the Sun, witnessing the beginning of the end of its host

https://celestia.mobi/addon?item=AEA8D485-8C2A-4F04-9E37-D7516448D3CB

**Comment by Weneedtocook** [comment]
Pretty cool.
